WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -7814,6 +7809,9 @@
 				sk,
 			);
 			mainGroup = 29B97314FDCFA39411CA2CEA /* CustomTemplate */;
+			packageReferences = (
```

Comment:
It seems Apple changed how you add local packages at some point. I updated it using the latest mechanism. It now seems to do a better job recognizing when something in the local modules changes.